be1-go/hub/standard_hub/mod.go

```diff
@@ -95,6 +95,12 @@ type Hub struct {
 
 	// peersGreeted stores the peers that were greeted by the socket ID
 	peersGreeted []string
+
+	// blacklist stores the IDs of the messages that failed to be processed by the hub
+	// the server will not ask for them again in the heartbeat
+	// and will not process them if they are received again
+	// @TODO remove the messages from the blacklist after a certain amount of time by trying to process them again
```

Comment:
there is a TODO here, i suppose it is left for future work?

Reply:
Yes !